docs/core/tools/csproj.md

```diff
@@ -140,7 +140,7 @@ A `<PackageReference>` item element specifies a [NuGet dependency in the project
 
 #### Version
 
-The required `Version` attribute specifies the version of the package to restore. The attribute respects the rules of the [NuGet versioning](/nuget/reference/package-versioning#version-ranges-and-wildcards) scheme. The default behavior is an exact version match. For example, specifying `Version="1.2.3"` is equivalent to NuGet notation `[1.2.3]` for the exact 1.2.3 version of the package.
+The required `Version` attribute specifies the version of the package to restore. The attribute respects the rules of the [NuGet versioning](/nuget/reference/package-versioning#version-ranges-and-wildcards) scheme. The default behavior is a minimum version, inclusive match. For example, specifying `Version="1.2.3"` means the resolved package will have the version 1.2.3 if available or greater otherwise.
```

Comment:
Per suggestion from @nkolev92 

```suggestion
The required `Version` attribute specifies the version of the package to restore. The attribute respects the rules of the [NuGet versioning](/nuget/reference/package-versioning#version-ranges-and-wildcards) scheme. The default behavior is a minimum version, inclusive match. For example, specifying `Version="1.2.3"` is equivalent to specifying `Version="[1.2.3, )"` and means the resolved package will have the version 1.2.3 if available or greater otherwise.
```